locallib.php

```diff
@@ -0,0 +1,61 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Functions and classes for block management
+ *
+ * @package    block
+ * @subpackage coursefeedback
+ * @copyright  2023 Technische Universität Berlin
+ * @author     2011-2023 onwards Jan Eberhardt
+ * @author     2023 onwards Felix Di Lenarda
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+/**
+ * Adds or removes the coursefeedbackblock in all courses
+ */
+function install_and_deinstall_block() {
```

Comment:
install_and_remove_block. "Deinstall" ist ganz schön schräges Englisch und habe ich in diesem Kontext noch nie gelesen.

Reply:
Haha engl. 2 Punkte damals :D
[2bf4914](https://github.com/eberhardt/moodle-block_coursefeedback/pull/18/commits/2bf491427140c93a4d82288908052b79f6a3355b)

Reply:
^^
Ansonsten eine schöne Idee. Die Umsetzung sehr viel besser als ich das damals gemacht habe. :D